From: Julian Andres Klode <jak@debian.org>
Date: Thu, 10 Apr 2014 16:17:39 +0200
Subject: Set /boot/efi as default ESP mount point

The Debian installer configures the ESP to be mounted at /boot/efi,
and not /boot, as gummiboot assumes.

Forwarded: not-needed
---
 src/setup/setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/setup/setup.c b/src/setup/setup.c
index 6a4275a..0cfaccf 100644
--- a/src/setup/setup.c
+++ b/src/setup/setup.c
@@ -1367,7 +1367,7 @@ int main(int argc, char*argv[]) {
         }
 
         if (!arg_path)
-                arg_path = "/boot";
+                arg_path = "/boot/efi";
 
         if (geteuid() != 0) {
                 fprintf(stderr, "Need to be root.\n");
